src/NServiceBus.Transport.RabbitMQ.TransportTests/NServiceBus.Transport.RabbitMQ.TransportTests.csproj

```diff
@@ -11,11 +11,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="[15.*,16)" />
-    <PackageReference Include="NServiceBus" Version="[7.0.0-*,8)" />
-    <PackageReference Include="NServiceBus.TransportTests.Sources" Version="[7.0.0-*,8)" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="NServiceBus" Version="7.0.0-beta0007" />
+    <PackageReference Include="NServiceBus.TransportTests.Sources" Version="7.0.0-beta0007" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
```

Comment:
We seem to be missing the explicit NUnit reference in this project, which means we'll be getting whatever the lowest one that meets the NServiceBus.TransportTests.Sources dependency instead.

Reply:
I was going to say that's OK because we don't actually consume NUnit explicitly in the project (only the imported tests do), but on the other hand, we've seen issues with versions of NUnit3TestAdapter and we're fixing that to a specific version, so we really need to fix NUnit to a specific version which we know works with that version of the adapter, so I think it's better to have an explicit NUnit reference. I'll add it.